submissions/adeyemiJuwon/javascript/loop.js

```diff
@@ -0,0 +1,19 @@
+let a = 123;
+  let b = 7;
+  let c = a - b;
+
+for (var i = 1; i < c; i++){
```

Comment:
it should be less than or equal to `c` @Samju200

Reply:
Okay
Done
Thank you @ToluAkin